src-electron/sdk/matter.js

```diff
@@ -118,33 +100,38 @@ async function getDisabledStorage(db, zcl) {
  * @returns Storage Option
  */
 
-async function computeStorageNewConfig(
-  db,
-  clusterRef,
-  storagePolicy,
-  forcedExternal,
-  attributeName
-) {
+async function computeStorageOptionNewConfig(storagePolicy) {
   let storageOption
-  let clusterName
-  clusterName = await queryCluster.selectClusterName(db, clusterRef)
   if (storagePolicy == dbEnum.storagePolicy.attributeAccessInterface) {
     storageOption = dbEnum.storageOption.external
   } else if (storagePolicy == dbEnum.storagePolicy.any) {
     storageOption = dbEnum.storageOption.ram
   } else {
     throw 'check storage policy'
   }
-  if (
-    forcedExternal.byName &&
-    forcedExternal.byName[clusterName] &&
-    forcedExternal.byName[clusterName].includes(attributeName)
-  ) {
-    storageOption = dbEnum.storageOption.external
-  }
   return storageOption
 }
 
+async function computeStoragePolicyNewConfig(
+  db,
+  clusterRef,
+  storagePolicy,
+  forcedExternal,
+  attributeName
+) {
+  let clusterName
+  clusterName = await queryCluster.selectClusterName(db, clusterRef)
+  forcedExternal.map((option) => {
```

Comment:
.map may be a typo here? Convert to for loop and break out of it once the if condition is satisfied?

Reply:
Unresolving this since it looks the same. Needs a forEach?